CONTRIBUTING.md

```diff
@@ -17,44 +17,77 @@ with the tutorial.
-  - [astropy.coordinates](http://docs.astropy.org/en/stable/coordinates/)
-  - [astroquery](https://astroquery.readthedocs.io/en/latest/)
-
+    - use [astropy.visualization](http://docs.astropy.org/en/stable/visualization/) where possible
+    
 Code:
 - Demonstrate good commenting practice
+  - add comments to bits of code which use concepts not included in Learning Goals
 - Demonstrate best practices of variable names. 
    - Variables should be all lower case with words separated by underscores.
    - Variable names should be descriptive. E.g., galaxy_mass, u_mag.
+- Use the print function explicitly to display information about variables
 - As much as possible, comply with [PEP8](https://www.python.org/dev/peps/pep-0008/)
+- Imports
+  - Do not import `*`. Import package and functions explicitly. 
+  - Recommended import block and abbreviations
+    ```python
+    import numpy as np
+    import matplotlib as mpl
+    import matplotlib.pyplot as plt
+    import astropy.units as u
+    import astropy.coordinates as coord
+    from astropy.io import fits
+
+    %matplotlib inline # make plots display in notebooks
+    ```
+
+
+Narrative:
+- Please read through the other tutorials to get a sense of the desired tone and length. 
+- Use [Markdown formatting](http://jupyter-notebook.readthedocs.io/en/latest/examples/Notebook/Working%20With%20Markdown%20Cells.html) in text cells for formatting, links, latex, and code snippets. 
+- Title should be short and descrictive
+- List all author's full names (comma separated) and link to github profile and/or [ORCID](https://github.com/astropy/astropy-tutorials/pull/171) id.
+- Include [Learning Goals](http://tll.mit.edu/help/intended-learning-outcomes) at the top as a bulleted list.
+- Include Keywords as a comma separated list of topics, packages, and functions demonstrated.
+- The first paragraph should give a brief overview of the entire tutorial including relevant astronomy concepts.
+- Use the first-person inclusive plural ("we"). For example, "We are going to make a plot which..", "Above, we did it the hard way, but here is the easier  way..."
+- Section Headings should be in the imperative mood. For example, "Download the data."
+- Avoid words such as "obviously","just", "simply", "easily". For example, "we just have to do this one thing."
+- Use `<div class="alert alert-info">Note</div>` for Notes and `<div class="alert alert-warning">Warning</div>` for Warnings.
+
+Template:
+# Doing a thing with things
+
+## Authors
+Jane Smith, Jose Jones
+
+## Leaning Goals
+- Query..
+- Calculate..
+- Display..
 
-Description:
-- Compile a list of the functions and packages the tutorial demonstartes and include a short a description with the pull request.
+## Keywords
+Example, example, example
+
+## Companion Content
+Carroll & Ostlie 10.3, Binney & Tremian 1.5
+
+This tutorial goes from a downloading a data file, doing something to it, and visualizing it.
```

Comment:
"This tutorial goes from a downloading..." -> "This tutorial goes from downloading..."